sentinel-core/src/test/java/com/alibaba/csp/sentinel/eagleeye/EagleEyeCoreUtilsTest.java

```diff
@@ -0,0 +1,164 @@
+package com.alibaba.csp.sentinel.eagleeye;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class EagleEyeCoreUtilsTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testIsBlank() {
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(""));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(" "));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(null));
+
+        Assert.assertFalse(EagleEyeCoreUtils.isBlank("foo"));
+    }
+
+    @Test
+    public void testCheckNotNullEmpty() {
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.checkNotNullEmpty("foo", "bar"));
+
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNullEmpty("", "bar");
```

Comment:
I think an exception will be thrown in the first method call so you will need separate test methods for the other two.

Reply:
All 3 throws an exception. I have moved them to a separate method

Reply:
I understand that each of them on their own will throw an exception but I think the method exits once the first exception is thrown so the 2nd and 3rd would not actually get tested if in the same method.

Reply:
Ahhh i see what you mean.. I have moved them to 3 separate methods